internal/controllers/objectsetphases/objectsetphase_reconciler.go

```diff
@@ -15,7 +15,8 @@ import (
 
 	corev1alpha1 "package-operator.run/apis/core/v1alpha1"
 	"package-operator.run/internal/controllers"
-	"package-operator.run/internal/probing"
+	internalprobing "package-operator.run/internal/probing"
```

Comment:
I suppose we still have some code that we do not want to push to the submodule since we do not want to guarantee stability, right?